### PR TITLE
Misc corrections

### PR DESCRIPTION
**Pull request by symbolpunk**

### CODE REVIEW

File: website/versioned_docs/version-2.1.0/execution-node/fee-recipient.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Miners currently collect these priority fees. After The Merge, proof-of-work con
 Because priority fees are captured by execution clients in the execution layer, validator clients need to tell execution clients where to forward these priority fees. This priority fee “forwarding address” is referred to as your **fee recipient** address.
 
 :::tip Configure this before The Merge
-If you don't configure your fee recipient address before The Merge, your priority fee earnings will be deposited into a burn address (`0x000....0`).
+If you don't configure your fee recipient address before The Merge, your priority fee earnings will be deposited into a burn address (`0x0`).
```
Comment (james-prysm) [inline website/versioned_docs/version-2.1.0/execution-node/fee-recipient.md:27]: I guess we should have it as "0x0000000000000000000000000000000000000000"

File: website/versioned_docs/version-2.1.0/prysm-usage/slasher.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ These specifications must be met in order to successfully run a Prysm beacon nod
 * Operating System: 64-bit Linux, Mac OS X 10.14+, Windows 64-bit
 * Processor: Intel Core i5–760 or AMD FX-8100 or better
 * Memory: 8GB RAM
-* Storage: 1TB available space SSD
+* Storage: 200GB available space SSD
```
Comment (james-prysm) [inline website/versioned_docs/version-2.1.0/prysm-usage/slasher.md:21]: is this accurate? 🤔

Comment (symbolpunk) [reply website/versioned_docs/version-2.1.0/prysm-usage/slasher.md:21]: updated ty! (todo: improve the way we do versioning / syncing between version directories in docusaurus....)

**Review by james-prysm — APPROVED** [review]
LGTM

**Pull request merged**